source/common/upstream/cluster_manager_impl.cc

```diff
@@ -1265,6 +1277,53 @@ ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::~ClusterEntry()
   }
 }
 
+Http::ConnectionPool::Instance*
+ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::getHttpPool(
```

Comment:
@mattklein123 This is the follow up when both the host. Problem solved if we put this in the interface of ClusterEntry.